web/redux/actions/active-rooms-actions.js

```diff
@@ -0,0 +1,45 @@
+import API from '../../api';
+
+import * as actions from '../constants/active-rooms-contants';
+
+
+export function loadingMoreActiveRooms() {
+  return { type: actions.ACTIVE_ROOMS_LOADING_MORE };
+}
+
+export function loadedMoreActiveRooms(activeRooms) {
+  return { type: actions.ACTIVE_ROOMS_LOADED_MORE, activeRooms };
+}
+
+export function failedLoadingMoreActiveRooms(error) {
+  return { type: actions.ACTIVE_ROOMS_FAILED_LOADING_MORE, error };
+}
+
+export function setActiveRoomsComplete() {
+  return { type: actions.ACTIVE_ROOMS_COMPLETE };
+}
+
+export function loadMoreActiveRooms() {
```

Comment:
By convention in this project, actions that return functions start with `handle` so it's clear there's work being done to callers. So can you rename this to `handleLoadMoreActiveRooms`?